records_mover/records/targets/table/move_from_fileobjs_source.py

```diff
@@ -46,6 +49,10 @@ def reset_before_reload(self) -> None:
     def move(self) -> MoveResult:
         with self.tbl.db_engine.begin() as db:
             driver = self.tbl.db_driver(db)
+            # TODO: Can this take in a loader so we don't have to do this assertion here?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker